syntax = "proto3";
option java_multiple_files = true;
option java_package = "com.github.saturn_xiv.palm.plugins.morus.v1";
package palm.morus.v1;

// ----------------------------------------------------------------------------
service Markdown {
  rpc ToHtml(MarkdownToHtmlRequest) returns (MarkdownToHtmlResponse) {}
}

message MarkdownToHtmlRequest {
  string payload = 1;
}
message MarkdownToHtmlResponse {
  string payload = 1;
}
